repository/test-src/org/pentaho/platform/repository2/unified/webservices/jaxws/UnifiedRepositoryToWebServiceAdapterIT.java

```diff
@@ -75,7 +78,7 @@ public void testFileMetadata() throws Exception {
       assertNotNull( testfile.getId() );
       final Map<String, Serializable> fileMetadata = repo.getFileMetadata( testfile.getId() );
       assertNotNull( fileMetadata );
-      assertEquals( 0, fileMetadata.size() );
+      assertEquals( 1, fileMetadata.size() );
```

Comment:
@mbatchelor The "schedulable" flag it's a new flag for FileMetadata. I store the "schedulable" value there.